src/main/java/de/uhd/ifi/se/decision/management/jira/rest/impl/KnowledgeRestImpl.java

```diff
@@ -87,8 +88,8 @@ public Response getUnlinkedElements(@QueryParam("id") long id, @QueryParam("proj
 	@Path("/createDecisionKnowledgeElement")
 	@POST
 	@Produces({ MediaType.APPLICATION_JSON })
-	public Response createDecisionKnowledgeElement(@Context HttpServletRequest request,
-			KnowledgeElement element, @QueryParam("idOfExistingElement") long idOfExistingElement,
+	public Response createDecisionKnowledgeElement(@Context HttpServletRequest request, KnowledgeElement element,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'createDecisionKnowledgeElement(HttpServletRequest, KnowledgeElement, long, String, String)' has an NPath complexity of 432](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=4882475)